### DISCUSSION TITLE
Legacy documentation

### DISCUSSION THREAD

**Discussion by Jaimeloeuf** [firebase/firebase-admin-node#1474] [Q&A]
Hi maintainers, following the new v10 release, I cant seem to find the documentation for v9 anywhere, can someone please help to point me to it? Thanks alot!

**Comment by lahirumaramba** [comment]
Hi @Jaimeloeuf, we update our reference docs with every new release. Since the V10 release supports both the modular and namespaced API, you can access the legacy API docs [here](https://firebase.google.com/docs/reference/admin/node/firebase-admin). Hope that helps.
